packages/middleware/src/middleware.ts

```diff
@@ -195,12 +197,7 @@ export interface MiddlewareError {
 
 export type FinalBody = Package | MiddlewareError | string;
 
-export function final(
-  body: FinalBody,
-  req: $RequestExtend,
-  res: $ResponseExtend,
-  next: $NextFunctionVer
```

Comment:
You are removing `next`, that explain why all test are failing. This is being used in that method.

Reply:
Oh... eslint was complaining this as unused var. I will revert this change.